tests/src/rules/t-call-in-function.test.ts

```diff
@@ -34,7 +34,13 @@ ruleTester.run<string, readonly unknown[]>('t-call-in-function', rule, {
                 padding: ${tokens.spacing20};\
               `',
     },
+    {
+      code: 'const a = () => {t("Hello")}',
+    },
   ],
 
-  invalid: [{ code: 't`Hello`', errors }],
+  invalid: [
+    { code: 't`Hello`', errors },
+    { code: 't("Hello")', errors },
+  ],
```

Comment:
there are also signatures where you can pass an object as a param, add them to the test please just to be sure that rule not expecting usage with Literal only

Reply:
sure